metricbeat/module/zookeeper/_meta/kibana/7/dashboard/Metricbeat-zookeeper-overview.json

```diff
@@ -0,0 +1,504 @@
+{
```

Comment:
I don't think this is going to work as in 6.x that 6 directory is needed?

Please try to import the dashboard against a 6.x stack setup to make sure things work.